dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessDefinitionMapper.xml

```diff
@@ -172,7 +172,7 @@
         FROM t_ds_process_task_relation ptr
         join t_ds_process_definition pd
             on ptr.process_definition_code=pd.code and ptr.process_definition_version = pd.version
-                and ptr.project_code=pd.project_code and  pd.release_state = 1
+                and ptr.project_code=pd.project_code
```

Comment:
But I found `processDefinitionMapper.listResources()` is just called by UT, please check it.

Reply:
This problem only exists in 3.0.x, the code in dev is different.